contracts/schemes/ReputationFromToken.sol

```diff
@@ -38,6 +39,9 @@ contract ReputationFromToken {
         require(redeems[msg.sender] == false, "redeeming twice from the same account is not allowed");
         redeems[msg.sender] = true;
         uint256 tokenAmount = tokenContract.balanceOf(msg.sender);
+        if (_beneficiary == address(0)) {
+            _beneficiary = msg.sender;
```

Comment:
why is this needed ?
if it is needed , please add test.

Reply:
0 is default value, we should handle it if it is called by mistake, and also allow call with no value.

Reply:
no issue. 
though other fault values can be submitted as well.